rc_django/views.py

```diff
@@ -78,9 +75,9 @@ def proxy(request, service, url):
 
     start = time()
     try:
-        if service == "iasystem" and subdomain is not None:
-            response = dao.getURL(url, headers, subdomain)
-
+        if re.match(r'^iasystem_', service):
+            headers = {"Accept": "application/vnd.collection+json"}
```

Comment:
Just in case other headers have already been set, you should use:

headers["Accept"] = "application/vnd.collection+json"

Reply:
Thank you for pointing that out!